package.json

```diff
@@ -5,7 +5,8 @@
   "main": "lib/ts-tinkerpop.js",
   "scripts": {
     "test": "make test",
-    "install": "make install-maven"
+    "install": "make install-maven",
+    "postpublish": "./devbin/update-definitely-typed.sh"
```

Comment:
I'm not sure this makes complete sense as a postpublish.  There are times when you want to update TSD on a topic branch to do integration testing, and you would want to use ts-tinkerpop from a URL (pre-publish).  But we can leave it for now.